test/unittests/execute_numeric_test.cpp

```diff
@@ -444,10 +446,10 @@ TEST(execute_numeric, i32_rotr)
 TEST(execute_numeric, i32_wrap_i64)
 {
     // <=32-bits set
-    EXPECT_THAT(execute_unary_operation(Instr::i32_wrap_i64, 0xffffffff), Result(0xffffffff));
+    EXPECT_THAT(execute_unary_operation(Instr::i32_wrap_i64, 0xffffffff_u64), Result(0xffffffff));
     // >32-bits set
     EXPECT_THAT(
-        execute_unary_operation(Instr::i32_wrap_i64, 0xffffffffffffffff), Result(0xffffffff));
+        execute_unary_operation(Instr::i32_wrap_i64, 0xffffffffffffffff_u64), Result(0xffffffff));
```

Comment:
Hmm, perhaps result could be also using the `_u32`/`_u64` literals.

Reply:
This is separate issue being handled in https://github.com/wasmx/fizzy/pull/659.